apps/storefront-e2e/src/support/page_fragments/header.fragment.ts

```diff
@@ -56,6 +56,8 @@ export class HeaderFragment {
     // eslint-disable-next-line cypress/no-unnecessary-waiting
     cy.wait(500);
     this.getCurrencyButton().click();
-    this.getCurrencySelector().find(`oryx-option[value="${currency}"]`).click();
+    this.getCurrencySelector()
+      .find(`oryx-option[value="${currency}"]`)
+      .click({ force: true });
```

Comment:
Is it safe? Is I understand we are ignoring every possible accessibility check, so this element can be invisible, disabled and so on.

Reply:
It is not safe for sure, but it is temporary
I'll remove this check when https://spryker.atlassian.net/browse/HRZ-3157 is fixed (added a note is the bug)